DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/HostApp/SpeckleDUI3Wrapper.cs

```diff
@@ -0,0 +1,16 @@
+using System.Windows.Controls;
+
+namespace Speckle.Connectors.ArcGIS.HostApp;
+
+public class SpeckleDUI3Wrapper : UserControl
+{
+  public SpeckleDUI3Wrapper()
+  {
+    Initialize();
+  }
+
+  private void Initialize()
+  {
+    Content = SpeckleModule.Container.Resolve<SpeckleDUI3>();
```

Comment:
Have you managed make it work like this?

Reply:
Working